src/update-chk.js

```diff
@@ -0,0 +1,16 @@
+function updateCheckbox(list, array, checked) {
+  array[Number(list.id)].completed = checked;
+  localStorage.setItem('data', JSON.stringify(array));
+  if (checked) {
+    list.classList.add('active');
+  } else {
+    list.classList.remove('active');
+  }
+}
+export function clearAllCompleted(array) {
```

Comment:
[OPTIONAL] Since you use ES6, it would be nicer to use arrow functions instead of the `function` word.